sigopt/xgboost/experiment.py

```diff
@@ -181,14 +183,31 @@ def run_experiment(self):
           self.dtrain,
           num_boost_round=num_boost_round_run,
           evals=self.evals,
+          early_stopping_rounds=self.early_stopping_rounds,
           verbose_eval=False,
           run_options=self.run_options,
         )
 
 
-def experiment(experiment_config, dtrain, evals, params, num_boost_round=None, run_options=None):
+def experiment(
+  experiment_config,
+  dtrain,
+  evals,
+  params,
+  num_boost_round=None,
+  run_options=None,
+  early_stopping_rounds=10,
```

Comment:
flip the order of `early_stopping_rounds` and `run_options`?

Reply:
done